src/components/TransactionsFees/index.tsx

```diff
@@ -46,7 +44,7 @@ export const TransactionFees = ({
         <Paragraph size="lg" align="center">
           You&apos;re about to {transactionAction} a transaction and will have to confirm it with your currently
           connected wallet.{' '}
-          {!isOffChainSignature && (
+          {!isOffChainSignature && isExecution && (
```

Comment:
Only checking for `isExecution` should be sufficient as it is mutually exclusive to `isOffChainSignature`

Reply:
I guess this should actually be `!isOffChainSignature || isExecution`. If a signature is needed but it cannot be off-chain, you still have to pay gas.

Reply:
I've adjusted the condition, so I'll resolve this thread if you don't mind.